src/main/java/bio/terra/service/snapshot/flight/delete/DeleteSnapshotMetadataStep.java

```diff
@@ -24,8 +23,7 @@ public DeleteSnapshotMetadataStep(SnapshotDao snapshotDao, UUID snapshotId) {
 
   @Override
   public StepResult doStep(FlightContext context) {
-    Snapshot snapshot = null;
-    boolean found = false;
+    boolean found;
```

Comment:
Not sure why this code doesn't use `DeleteResponseModel.ObjectStateEnum` here instead of `boolean`. Using `boolean` here means that we have to convert it to the object state later.